posts/Eventsource.md

```diff
@@ -1,7 +1,12 @@
 feature: EventSource (Server-Sent Events)
 status: use
-tags: fallback
+tags: polyfill
 kind: api
-polyfillurls:[EventSource.js](https://github.com/remy/polyfills/blob/master/EventSource.js), [jQuery.eventsource](https://github.com/rwldrn/jquery.eventsource)
+polyfillurls:[EventSource polyfill](https://github.com/Yaffle/EventSource)
 
-[Server-Sent Events](http://html5doctor.com/server-sent-events/) are real-time events emitted by the server and received by the browser. EventSource.js provides a base fallback and is not dependent on jQuery.
+Server-sent events are real-time events emitted by the server and received by the browser.
+The main different to WebSockets is that they work over regular HTTP and can therefore be replicated entirely using JavaScript. 
+
+Event Source isn’t available in any version of Internet Explorer and still under [consideration for Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/serversenteventseventsource/?q=EventSource).
```

Comment:
Should Event Source be EventSource here?

Reply:
Your right.